collectors/ebpf.plugin/ebpf_dcstat.c

```diff
@@ -160,7 +160,7 @@ void ebpf_dcstat_create_apps_charts(struct ebpf_module *em, void *ptr)
 {
     struct target *root = ptr;
     ebpf_create_charts_on_apps(NETDATA_DC_HIT_CHART,
-                               "Percentage of files listed inside directory cache",
+                               "Percentage of files inside directory cache",
```

Comment:
```suggestion
                               "Percentage of files inside the directory cache",
```
There are other updates without "the" article.  Obviously adding it is not crucial but just a suggestion.